cypress/integration/sourcepoint-tcfv2.spec.js

```diff
@@ -57,7 +55,7 @@ skipOn(Cypress.env('CI') === 'true', () => {
 				.find(`button[title="${buttonTitle}"]`)
 				.click();
 
-			// eslint-disable-next-line cypress/no-unnecessary-waiting
+			// eslint-disable-next-line cypress/no-unnecessary-waiting -- should we do this?
```

Comment:
I think we can wait for a cookie to be present… let’s look at how it’s done in CCPA and raise another PR.